packages/okidoc-md/src/buildMarkdown/buildMarkdown.spec.js

```diff
@@ -365,15 +377,16 @@ describe('buildMarkdown', () => {
 
     it('should render valid markdown with function in return type interface', async () => {
       const documentationSource = `
-        /**
-        * @property update - update entity
-        * @property destroy - destroy entity
-        */
+        /** */
         interface MyEntity {
+          /** */
```

Comment:
without `/** */` it will not add to doc result?

Reply:
Yes